Banyan/src/clusters.jl

```diff
@@ -1,6 +1,24 @@
+struct Cluster
+    name::String
+    status::Symbol
+    status_explanation::String
+    s3_bucket_arn::String
+    organization_id::String
+    curr_cluster_instance_id::String
+    num_sessions_running::Int64
```

Comment:
Why are these stored here on the client side? They don't seem to be used anywhere.

Reply:
Talked about this in private slack!